src/super_gradients/training/datasets/detection_datasets/detection_dataset.py

```diff
@@ -74,6 +74,9 @@ def __getitem__(self, index):
         if self.sample_loading_method == 'mosaic' and self.augment:
             # LOAD 4 IMAGES AT A TIME INTO A MOSAIC (ONLY DURING TRAINING)
             img, labels = self.load_mosaic(index)
+            # MixUp augmentation
+            if random.random() < self.dataset_hyperparams['mixup']:
```

Comment:
wouldn't this crash if  self.dataset_hyperparams['mixup'] is not set?

Reply:
right, fixed setting default mixup prob to 0.